pyproject.toml

```diff
@@ -15,8 +15,9 @@ testpaths = ["tests"]
 filterwarnings = [
     "error",
     "ignore:^.*pkg_resources.*:DeprecationWarning",
-    "ignore::sqlalchemy.exc.MovedIn20Warning"
+    "ignore::sqlalchemy.exc.MovedIn20Warning",
+    "ignore:^.*You are trying to use a chat model. This way of initializing it is no longer supported. Instead, please use.*:UserWarning",
```

Comment:
LangChain prefers a different way of initializing OpenAI chat models now. The old one still works however.

Reply:
It seems like you did modify the instantiation method though... Should we keep the last exception?

Reply:
No, I'm just passing `model_name` as separate arg. What LangChain actually wants us to do is to instantiate a differently named class. The current approach does exactly the same behind the LangChain curtains, but with a warning and rerouting to this new class with a warning.

What we should actually do is to figure out a new way how to instantiate the correct `LangChain` classes generically (like it's done currently with `langchain.llms.type_to_cls_dict`. I haven't looked into this and would do so in a separate PR.